src/animation/friction.ts

```diff
@@ -66,79 +64,107 @@ function reversePlayState(
   }
 }
 
+function checkFrictionStoppingCondition({
+  state,
+}: StatefulFrictionAnimatingElement) {
+  return state.mover.velocity[0] <= 0;
+}
+
+let isFrameloopActive = false;
+const animatingElements = new Set<StatefulFrictionAnimatingElement>();
```

Comment:
Here's the money marker right here, an idea taken from `react-spring`'s implementation: https://github.com/react-spring/react-spring/blob/master/src/animated/FrameLoop.ts We store all animating elements in a `Set`. When the function below is called, we iterate through all nodes handed to it and add them to the `Set`. `controller.stop` then performs the action of removing elements from the `Set` when they're no longer animated. Beautifully simple!